apps/meteor/client/views/room/hooks/useUserInfoActions/actions/useAddUserAction.tsx

```diff
@@ -0,0 +1,89 @@
+import type { IRoom, IUser } from '@rocket.chat/core-typings';
+import { isRoomFederated } from '@rocket.chat/core-typings';
+import { useEffectEvent } from '@rocket.chat/fuselage-hooks';
+import {
+	useTranslation,
+	useUser,
+	useUserRoom,
+	useUserSubscription,
+	useMethod,
+	useToastMessageDispatch,
+	useAtLeastOnePermission,
+} from '@rocket.chat/ui-contexts';
+import { useMemo } from 'react';
+
+import * as Federation from '../../../../../lib/federation/Federation';
+import { useAddMatrixUsers } from '../../../contextualBar/RoomMembers/AddUsers/AddMatrixUsers/useAddMatrixUsers';
+import { getRoomDirectives } from '../../../lib/getRoomDirectives';
+import type { UserInfoAction } from '../useUserInfoActions';
+
+export const useAddUserAction = (
+	user: Pick<IUser, '_id' | 'username'>,
+	rid: IRoom['_id'],
+	reload?: () => void,
+): UserInfoAction | undefined => {
+	const t = useTranslation();
+	const room = useUserRoom(rid);
```

Comment:
Use `useRoom` instead.

Reply:
![useRoom](https://github.com/RocketChat/Rocket.Chat/assets/15830206/09b1bce2-819a-4c99-ac56-1404a0b39982)
It seems like the UserCard is not rendered inside of Room context.
useUserRoom eliminates this error.